specification/kubernetesconfiguration/resource-manager/Microsoft.KubernetesConfiguration/preview/2021-10-01-preview/fluxconfiguration.json

```diff
@@ -0,0 +1,786 @@
+          },
+          {
+            "name": "fluxConfiguration",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/FluxConfiguration"
+            },
+            "description": "Properties necessary to Create a FluxConfiguration."
+          }
+        ],
+        "responses": {
+          "201": {
+            "description": "Request received successfully.",
+            "schema": {
+              "$ref": "#/definitions/FluxConfiguration"
+            }
+          },
+          "409": {
+            "description": "Conflict",
+            "x-ms-error-response": true,
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/ErrorResponse"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "x-ms-long-running-operation-options": {
+          "final-state-via": "azure-async-operation"
+        }
+      },
+      "patch": {
+        "tags": [
+          "FluxConfiguration",
+          "Flux"
+        ],
+        "description": "Update an existing Kubernetes Flux Configuration.",
+        "operationId": "FluxConfigurations_Update",
+        "x-ms-examples": {
+          "Patch Flux Configuration": {
+            "$ref": "./examples/PatchFluxConfiguration.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "../../common/2021-10-01-preview/parameters.json#/parameters/ClusterRpParameter"
+          },
+          {
+            "$ref": "../../common/2021-10-01-preview/parameters.json#/parameters/ClusterResourceNameParameter"
+          },
+          {
+            "$ref": "../../common/2021-10-01-preview/parameters.json#/parameters/ClusterNameParameter"
+          },
+          {
+            "$ref": "#/parameters/FluxConfigurationNameParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "name": "fluxConfigurationPatch",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/FluxConfigurationPatch"
+            },
+            "description": "Properties to Patch in an existing Flux Configuration."
+          }
+        ],
+        "responses": {
```

Comment:
>responses [](http://example.com/codeflow?start=9&length=9)

Recommended pattern is to include 200 as a possible response code to cover the synchronous case. You can support synchronous response in future that way.